update/docker-compose.uta.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  seqrepo:
```

Comment:
I don't think we need this seqrepo section of the docker compose file.

Reply:
Yes, I can delete

Reply:
Deleted in #9